app/models/Campaign.js

```diff
@@ -112,18 +94,47 @@ campaignSchema.statics.lookupByIDs = function (campaignIDs) {
  * @see https://github.com/DoSomething/gambit/issues/673
  */
 campaignSchema.methods.createMobileCommonsGroups = function () {
-  const campaign = this;
-  const prefix = `env=${process.env.NODE_ENV}
-                  campaign_id=${campaign._id}
-                  run_id=${campaign.current_run}`;
-
-  // Create mobile commons group with custom name based on this campaign
-  mobilecommons.createGroup(`${prefix} status=doing`)
-  .then(doingGroup => setMobileCommonsGroup(campaign, 'doing', doingGroup))
-  .then(() => mobilecommons.createGroup(`${prefix} status=completed`))
-  .then(completedGroup => setMobileCommonsGroup(campaign, 'completed', completedGroup))
-  .then(() => campaign.save())
-  .catch(err => logger.error(err));
+  const campaignId = this._id;
+  const campaignRunId = this.current_run;
```

Comment:
Minor question -- is it more common in JS to use `whateverId` over `whateverID`, or just a preference thing? I've been using the capital `IDs` -- i'm happy to Replace All the ID's in a future PR but let's pick one as rule to keep consistent.